xesmf/frontend.py

```diff
@@ -157,11 +157,16 @@ def ds_to_ESMFgrid(ds, need_bounds=False, periodic=None, append=None):
     else:
         mask = None
 
+    if 'pole_kind' in ds:
+        pole_kind = np.asarray(ds['pole_kind'])
```

Comment:
The possibility of passing `pole_kind` in the dataset should maybe be added somewhere in the documentation ? The docstring of `Regridder` maybe ?